src/apps/service/apps/de/jobs/common.clj

```diff
@@ -103,8 +73,9 @@
      component)
    :id))
 
-(defn- load-step-component
-  [task-id requirements]
+
+(defn- load-tool-info
+  [task-id]
```

Comment:
I split `load-tool-info` out from `load-step-component` for debugging, and I decided just to leave it that way.